frontend/src/components/browse-sightings-page/marker.tsx

```diff
@@ -0,0 +1,19 @@
+﻿import React from "react";
+
+interface MarkerProps {
+  text: string;
+  lat: number;
+  lng: number;
+}
+
+export const Marker: React.FC<MarkerProps> = ({ text }) => {
+  // const handleClick = () => {
+  //   console.log(You clicked on ${tooltip});
+  // };
+
+  return (
+    <div className="circle">
```

Comment:
"Circle" and "Circle text" aren't specific enough class names I'm afraid, I think `sighting-map__marker` is best!

Reply:
Also you don't actually put any styling on this component at all, and it only contains text, so I'm confused as to exactly why this needs to be a component at all instead of just passing a bunch of strings in?